pennylane/measurements/measurements.py

```diff
@@ -313,7 +313,7 @@ def __repr__(self):
             return f"{self.return_type.value}(eigvals={self._eigvals}, wires={self.wires.tolist()})"
 
         # Todo: when tape is core the return type will always be taken from the MeasurementProcess
-        return f"{self.return_type.value}(wires={self.wires.tolist()})"
+        return f"{getattr(self.return_type, 'value', 'None')}(wires={self.wires.tolist()})"
```

Comment:
what required this change? does any measurement actually have the repr `"None(wires=[])"`?

Reply:
This test
https://github.com/PennyLaneAI/pennylane/blob/c7c2b3d5ff8f24860e52f9f74395a9dd86926d5e/pennylane/devices/tests/test_measurements.py#L1761

was causing issues when the error message was being formatted.